### PR TITLE
Update AHM pages with virtual announcement

### PR DESCRIPTION
**Pull request by djw8605**
Delete hotels and reschedule pages.

### CODE REVIEW

File: docs/2020/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 <div style="border: 1px solid #3F51B5; color: #20295A; background-color: #E7E9F6; padding: 1ex; font-size: 115%; font-weight: bold;">
   31 August&nbsp;&ndash;&nbsp;3 September 2020
   <br>
-  The University of Oklahoma, Norman, OK
+  Virtual All-Hands Meeting
 </div>
 
-The detailed agenda will be posted later, check back here.
+The detailed agenda will be posted in July, check back here.
```
Comment (osg-cat) [inline docs/2020/index.md]: The *detailed agenda* may not be ready in July....... Just the block schedule (overview). I think “later” is fine right now.

File: docs/2020/registration.md [hunk 2]
```diff
@@ -1,9 +1,5 @@
 # OSG All-Hands Meeting 2020 &ndash; Registration
 
-## Costs
+Registration is free, but required.
```
Comment (osg-cat) [inline docs/2020/registration.md]: Participation in the event will be free, but we plan to ask participants to register in advance.

File: docs/2020/registration.md [hunk 3]
```diff
@@ -1,9 +1,5 @@
...
+We plan to have a link to the registration system when ready.
```
Comment (osg-cat) [inline docs/2020/registration.md]: The registration link will be posted here when available.

File: docs/2020/schedule.md [hunk 4]
```diff
@@ -12,4 +12,8 @@ September 3 ([rescheduled](/2020/reschedule) from March).
 
 ## High-Level Schedule
 
-Coming in late spring or early summer.
+Generally, the meetings will occupy just 4 to 5 hours per day, with breaks,
+so there will be time to attend and to do other things. We are aiming to
+have similar (but updated!) content as planned for March.
+
+Detailed schedule coming in July 2020.
```
Comment (osg-cat) [inline docs/2020/schedule.md]: We plan to post an overview schedule in July.

**Review by osg-cat — CHANGES_REQUESTED** [review]
Overall this looks really good and is very helpful, thank you! Just a few minor wording tweaks…

**Comment by djw8605** [comment]
Resolved comments.

**Review by osg-cat — APPROVED** [review]

**Pull request merged**